src/lib/baseCommand.ts

```diff
@@ -86,13 +88,23 @@ export default class Command {
    */
   args: OptionDefinition[];
 
-  run(opts: CommandOptions<{}>): Promise<string> {
+  async run(opts: CommandOptions<{}>): Promise<string> {
     Command.debug(`command: ${this.command}`);
     Command.debug(`opts: ${JSON.stringify(opts)}`);
 
     if (this.args.some(arg => arg.name === 'key')) {
       if (!opts.key) {
-        throw new Error('No project API key provided. Please use `--key`.');
+        if (isCI()) {
+          throw new Error('No project API key provided. Please use `--key`.');
+        } else {
+          // eslint-disable-next-line no-console
+          console.info("Looks like you're missing a ReadMe API key, let's fix that! 🦉");
```

Comment:
Don't we have a console helper somewhere to log out info statements?

Reply:
Good call! Our `info` helper prefixes all statements with a ℹ️ emoji but I extended that helper with a flag to configure that emoji prefix and refactored a bunch of `console.info` statements to use this: https://github.com/readmeio/rdme/commit/bab855db842e095fe6d521acd094607fbc4b33a3